external/loaders/tests/test__config.py

```diff
@@ -210,14 +208,6 @@ def test_load_batches_from_mapper_raises_if_registered_with_wrong_decorator():
 @pytest.mark.parametrize(
     "data, expected_class",
     [
-        pytest.param(
```

Comment:
Should we replace this case with a working example of a `BatchesConfig`? Otherwise it's hard to tell what this test is trying to do